bin/yarn-install

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
-ObservableProcess = require('observable-process')
+
+const ObservableProcess = require('observable-process'),
```

Comment:
In ES6 you have to provide `const` or `var`, otherwise the variable exists in the global scope.